api/src/main/resources/workbench-api.yaml

```diff
@@ -13141,6 +13141,11 @@ components:
     InitialCreditResponse:
       type: object
       properties:
+        expirationBypassed:
+          type: boolean
+          default: false
+          description: When set to true, the user will not be required to spend their
```

Comment:
so for BYO users, this will be `false`...can we add it in comment just to be explicit about it

Reply:
My understanding is that these are not directly connected.

Initial credits expire independently of BYO.

Reply:
Users could have had their expiration bypassed, used all their credits, and then swapped to BYO.

Similarly if they are not bypassed, the credits could have expired and then they switched to BYO.